### DISCUSSION TITLE
Augur model results

### DISCUSSION THREAD

**Discussion by inunezfe** [NREL/ReEDS-2.0#153] [Q&A: Analysis]
Good afternoon ReEDS team

I have run a separated interconnection with a new region that I created joining PJM and MISO regions but currently i am having issues with the retail module and the Augur module.

First of all, i am trying to debug the retail module and i have been running it cell by cell in visual studio code and i am getting this error after fixing most of the indexes on the other functions

![image](https://github.com/NREL/ReEDS_OpenAccess/assets/111766958/0606f055-26c7-4c29-8743-dcdae6134df9)

(example of indexes changed in order to be consistent and avoid python indexes errors)

![image](https://github.com/NREL/ReEDS_OpenAccess/assets/111766958/3666c97b-ee0d-45d7-8990-2725086a4083)

(actual error recevied by the retail_eval_period.h5 file)

That error is coming from this part of the code that reads this files with a .h5 extension

![image](https://github.com/NREL/ReEDS_OpenAccess/assets/111766958/bd8075b3-1020-4651-90e3-fa21d515f4cc)

after looking at those files there is not column called 'r' or 'region', just this data is shown or available. How can i move forward or fix the retail rate module?

![image](https://github.com/NREL/ReEDS_OpenAccess/assets/111766958/9bb338ed-83eb-487a-aa01-ac07c70f256a)

Second, after running the case file, the augur module generates some plots, and one of those plots is a dispatch curve or a merit order curve as you put it in the code, i notice that in the curves that i got there is no renewable energy price, how can i include into this curve?

![image](https://github.com/NREL/ReEDS_OpenAccess/assets/111766958/b338c834-075b-41ea-9389-af395ff31876)

Thanks!

Let me know if you have any questions regarding the new region or the casefile

**Comment by wesleyjcole** [comment]
For the retail rate module fix, you can just remove the reference to 'region':

![image](https://github.com/NREL/ReEDS_OpenAccess/assets/49044852/671386c1-0a02-4a1c-b470-c2cd55968346)

You'll need to do the same thing for depreciation_sch.

For the merit order curve, there are renewable energy technologies (biopower, hydro).  Wind and solar technologies, which have zero marginal costs, are not explicitly represented in this dispatch portion of the Augur module.  Rather, the generators are dispatched against the net load (load minus wind and solar generation).  Therefore the merit order curve is for meeting the net load, not the overall load.
